src/components/Img.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./Img.css";
+
+const Img = ({ url, alt }) => {
```

Comment:
You have Image and Img components, is this on purpose?

Reply:
I know, I wanted to ask you about that. They do the same, but I don't know how to set up dimensions in all 3 cases...So I used Img for AuctionItem, and Image for picture in IntroductionAndConclusion. I've been meaning to ask you how to do that. The problem is thah the height in AuctionItem should be 350px, but in IntroductionAndConclusion it should be smaller. It occurred to me to check which component uses Image, and based on that to apply different styles, but it doesn't make any sense to me.

Reply:
You should let the parent decide how the image behaves. For example your Image component should take up all the available space of it's container. For example, if you put a container of 350px then the image should adjust itself to it.